sway/commands/exec_always.c

```diff
@@ -51,7 +52,36 @@ struct cmd_results *cmd_exec_always(int argc, char **argv) {
 	if ((pid = fork()) == 0) {
 		// Fork child process again
 		setsid();
+
 		if ((*child = fork()) == 0) {
+			// Acquire the current PATH
+			char* path = secure_getenv('PATH');
+			const char* extra_path = ':/usr/lib/sway';
```

Comment:
All of these strings need double quotes, did you even try to compile this before submitting it?

And review the style guide, you have numerous errors

Reply:
Yes, I got confused while amending the patch for style, and didn't rebuild right before resubmitting...